docs/multiplayer.md

```diff
@@ -117,6 +117,8 @@ Last updated:    2011-02-16
  - You can protect your server with a password via the console: 'set server_pw',
    or via the Start Server menu.
 
+- You can ban or kick problemantic players via the console: 'ban client_ip_address' or 'kick client_ip_address'
```

Comment:
This is misaligned with the other elements, i.e. add a single space before the `-` and it is good.
This line is much longer than any of the other lines, making it harder to read in a console environment. Maybe add a newline after the `:`?